tests/specifications/general_test.py

```diff
@@ -81,9 +83,17 @@ def test_check_037():
   for status, message in fval_results:
     assert RASTER_EXCEPTION_MESSAGE not in message
 
-  # and finaly, we make sure that there wasn't an ERROR
-  # which would mean FontValidator is not properly installed:
-  assert status != ERROR
+    # and finally, we make sure that there wasn't an ERROR
+    # which would mean FontValidator is not properly installed:
+    assert status != ERROR
+
+  # Simulate FontVal missing.
+  old_path = os.environ["PATH"]
+  os.environ["PATH"] = ""
+  with pytest.raises(OSError) as _:
+    status, _ = list(check(font))[-1]
```

Comment:
usage of `_` here looks a bit confusing/weird

keep it as `status, message` please